### PR TITLE
test(DCOS-14457): Integration test for header components

### PR DESCRIPTION
**Pull request by weblancaster**
This PR (DCOS-14457) adds integration test for header components where breadcrumb, header tab are included.

AC:
* When there are more than 3 breadcrumb items:
-Breadcrumbs should collapse so that only the last two breadcrumb items are visible
-An ellipsis should take place of all breadcrumbs except the last two
-Hovering the ellipsis should render a tooltip that displays the path that is hidden
-The Services icon should route back to the services overview

- When the viewport is narrow, the breadcrumbs should truncate and not overflow their container
- When the viewport is narrow, the page header actions should always be visible
- When a page header's tab is active, its activity indicator should be flush with the bottom of the page header.

**Checklist**
- [ ] Did you add a JIRA issue in a commit message or as part of the branch name?
- [ ] Did you add new unit tests?
- [x] Did you add new integration tests?
- [ ] If this is a regression, did you write a test to catch this in the future?

### CODE REVIEW

**Comment by weblancaster** [comment]
retest this please

**Comment by weblancaster** [comment]
retest this please

**Comment by Poltergeist** [comment]
run integration tests

File: tests/components/Header-cy.js [hunk 1]
```diff
@@ -0,0 +1,131 @@
+describe('Header Component', function () {
```
Comment (jfurrow) [inline tests/components/Header-cy.js]: Could we name this "Page Header Component" instead (and then rename the file to `PageHeader-cy`)? Just to be explicit about exactly which component we're testing.

**Comment by jfurrow** [comment]
Nice tests! I have one minor comment but otherwise this looks great!

File: tests/components/Header-cy.js (hunk 1 above)
Comment (weblancaster) [inline tests/components/Header-cy.js]: It's addressed @jfurrow thanks for the review 👍

**Comment by wavesoft** [comment]
Looks good, works great. Great tests @weblancaster !

**Pull request merged**